src/helpers/sharingHelper.js

```diff
@@ -507,7 +507,10 @@ module.exports = {
     }
     const apiURL = `apps/files_sharing/api/v1/shares/${shareID}`;
     const body = new URLSearchParams();
-    body.append("permissions", permissions);
+    const updatedPermissions = this.humanReadablePermissionsToBitmask(
+      permissions
+    );
+    body.append("permissions", `${updatedPermissions}`);
```

Comment:
```suggestion
    body.append("permissions", updatedPermissions);
```

Reply:
`append` takes value of string type only so I think it is better this way.